itests/direct_data_onboard_verified_test.go

```diff
@@ -873,14 +872,14 @@ func TestVerifiedDDOExtendClaim(t *testing.T) {
 	require.True(t, wait.Receipt.ExitCode.IsSuccess())
 
 	// Extend all claims with lower TermMax
-	msgs, err = cli.CreateExtendClaimMsg(ctx, client.FullNode, pcm, []string{}, verifiedClientAddr2, builtin.EpochsInYear*4, false, true)
+	msgs, err = cli.CreateExtendClaimMsg(ctx, client.FullNode, pcm, []string{}, verifiedClientAddr2, builtin.EpochsInYear*4, false, true, 100)
 	require.NoError(t, err)
 	require.Nil(t, msgs)
 
 	newclaim, err = client.StateGetClaim(ctx, miner.ActorAddr, verifreg.ClaimId(allocationId), types.EmptyTSK)
 	require.NoError(t, err)
 	require.NotNil(t, newclaim)
-	require.EqualValues(t, newclaim.TermMax, verifregtypes13.MaximumVerifiedAllocationTerm)
 
-	// TODO: check "claim-updated" message
```

Comment:
actually, would you mind leaving this TODO here, it's a note to me, and it should say "event" rather than "message"

Reply:
Sure